style.css

```diff
@@ -366,3 +366,110 @@ ul {
   font-weight: 500;
   color: #253858;
 }
+
+/* Contact Section */
+.contact-section {
+  background-color: white;
+  padding: 10vh 0 0 0;
+}
+
+.contact-container {
+  background-color: var(--blue);
+  padding: 15vh 8vw 5vh 8vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-top-left-radius: 8em;
+  background-image: url(assets/contact\ form\ shapes-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position-x: right;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.contact-container h2 {
+  font-weight: 700;
+  font-size: 2.5em;
+  line-height: 52px;
+  color: white;
+  margin: 0 0 8px 0;
+}
+
+.contact-container p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin-bottom: 50px;
+  text-align: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.labels {
+  display: none;
+}
+
+.input-field,
+.textarea-field {
+  width: calc(100% - 30px);
+  border-radius: 8px;
+  border: var(--input-border);
+  margin-bottom: 20px;
+  padding: 0 15px;
+  cursor: pointer;
+  font: var(--input-font);
+  color: var(--input-color);
+}
+
+.input-field {
+  height: 48px;
+}
+
+.textarea-field {
+  height: 170px;
+  margin-bottom: 22px;
+  padding: 15px;
+}
+
+.input-field::placeholder,
+.textarea-field::placeholder {
+  font: var(--input-placeholder);
+}
+
+.button-form {
+  align-self: self-start;
+  background-color: white;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 1em;
+  line-height: 24px;
+  color: var(--blue);
+  padding: 12px 16px;
+  border: 1px solid var(--blue);
+}
+
+.button-form:hover {
+  background-color: var(--blue);
+  color: white;
+  border-color: white;
+}
+
+.button-form:active {
+  background-color: #2230d2;
+  color: white;
+  border-color: #2230d2;
+}
```

Comment:
You did well  implementing the project requirement, please implement the following design changes so you'd have a better UI and better user interaction :

-  kindly include the background images for this contact form section to have a really pleasant form suitable for users' aesthetics
- kindly add the placeholder colors implement in the Figma design template
-  please adjust the submit button to be in the same margin as the other input just as it is in the design given
- kindly use the right font-color for the hover property of your button so it looks similar to the design we have 😄 

Your design :
![image](https://github.com/CVILLA09/Portfolio/assets/20537712/ed0ba382-2a10-4989-8c6c-06af4cc9ab4b)

Figma design :
![image](https://github.com/CVILLA09/Portfolio/assets/20537712/43f852b5-c4da-4f0e-9b87-11aec26a0ec1)